.rubocop.yml

```diff
@@ -0,0 +1,16 @@
+Style/Documentation:
+  Enabled: false
+
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles.
+Style/StringLiterals:
+  Enabled: false
+
+Metrics/LineLength:
+  Max: 80
```

Comment:
Good point, I'm not sure where that came from, probably inherited a file with 120 and switched it to 80 for my own use. I'll pull that out.